src/styles/shared/_page.scss

```diff
@@ -48,13 +48,26 @@ $actions-vertical-spacing: spacing(tight);
 @mixin page-header-layout {
   padding: spacing(loose) spacing(loose) 0;
 
+  &.newDesignLanguage {
+    padding: spacing() spacing() 0;
```

Comment:
Probably will need to do some work to skeleton, too, so it matches the new page design